bin/jarcheck

```diff
@@ -31,7 +31,10 @@ class Spec
 
   def add_source source
     puts "spec.add_source #{source}"
-    @sources << source
+    # Avoid duplicates
```

Comment:
I'd rather implement this as a single call to `.uniq` when `@sources` is complete.